### PR TITLE
WIP: net/wol: 2.0 release

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

**Comment by fichtner** [comment]
please set PLUGIN_REVISION=1 for incoming fixes. no time anymore today, but this can go in as a first step if you want

File: net/wol/src/opnsense/mvc/app/models/OPNsense/Wol/Wol.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
             <default></default>
             <filters>
                 <enable>/^(?!0).*$/</enable>
+                <if>/^(?!openvpn|ipsec|enc).*$/i</if>
```
Comment (fichtner) [inline net/wol/src/opnsense/mvc/app/models/OPNsense/Wol/Wol.xml]: @fabianfrz works, thanks! but I'll investigate if we could have a better fix to only include non-virtual ones as new ones could be added and this is static

Comment (fabianfrz) [reply net/wol/src/opnsense/mvc/app/models/OPNsense/Wol/Wol.xml]: @fichtner maybe add a flag to the interface field type

Comment (fichtner) [reply net/wol/src/opnsense/mvc/app/models/OPNsense/Wol/Wol.xml]: yupp, good idea... I'll look into it :)

**Pull request merged**